src/Http/Routing/WebRouting.php

```diff
@@ -64,59 +58,70 @@ public function wire(): void
         });
 
-        $this->router->controller(UserController::class)->middleware("admin")->group(function (): void {
-            $this->router->delete("/users/{user}", "destroy")->name("users.destroy");
-        });
-
-        $this->router->controller(MeetupController::class)->middleware("admin")->group(function (): void {
-            $this->router->get("/meetups", "index")->withoutMiddleware("admin")->name("meetups");
-            $this->router->get("/meetups/create", "create")->name("meetups.create");
-            $this->router->post("/meetups", "store")->name("meetups.store");
-            $this->router->get("/meetups/{meetup}/edit", "edit")->name("meetups.edit");
-            $this->router->put("/meetups/{meetup}", "update")->name("meetups.update");
-            $this->router->delete("/meetups/{meetup}", "destroy")->name("meetups.destroy");
-        });
-
-        $this->router->controller(OrganizationController::class)->middleware("admin")->group(function (): void {
-            $this->router->get("/organizations", "index")->withoutMiddleware("admin")->name("organizations");
-            $this->router->get("/organizations/create", "create")->name("organizations.create");
-            $this->router->post("/organizations", "store")->name("organizations.store");
-            $this->router->get("/organizations/{organization}/edit", "edit")->name("organizations.edit");
-            $this->router->put("/organizations/{organization}", "update")->name("organizations.update");
-            $this->router->delete("/organizations/{organization}", "destroy")->name("organizations.destroy");
-        });
-
-        $this->router->controller(OrganizationProfileController::class)->middleware("admin")->group(function (): void {
-            $this->router->get("/organizations/{organization}/profiles/create", "create")->name("organizations.profiles.create");
-            $this->router->post("/organizations/{organization}/profiles", "store")->name("organizations.profiles.store");
-            $this->router->get("/organizations/{organization}/profiles/{profile}/edit", "edit")->name("organizations.profiles.edit");
-            $this->router->put("/organizations/{organization}/profiles/{profile}", "update")->name("organizations.profiles.update");
-            $this->router->delete("/organizations/{organization}/profiles/{profile}", "destroy")->name("organizations.profiles.destroy");
+        $this->router->middleware("role:admin")->group(function (): void {
+            $this->router->controller(AdminController::class)->group(function (): void {
+                $this->router->get("/admin/dashboard", "dashboard")->name("admin.dashboard");
+                $this->router->get("/admin/users", "usersIndex")->name("admin.users");
+                $this->router->get("/admin/meetups", "meetupsIndex")->name("admin.meetups");
+                $this->router->get("/admin/organizations", "organizationsIndex")->name("admin.organizations");
```

Comment:
Only admin user can access to that resources?

Reply:
Maybe we should think about levels of access in our app?

Reply:
@kamilpiech97  At the moment it is only for the administrator. We haven't decided on other roles in our app yet. #15

Reply:
`Index` and `show` routes of other controllers are available for all users.